devex/vscode-extension/marketplace-homepage.md

```diff
@@ -0,0 +1,383 @@
+# Open Enclave extension for Visual Studio Code
+
+A
+[Visual Studio Code](https://code.visualstudio.com/)
+[extension](https://marketplace.visualstudio.com/VSCode)
+supporting
+[Open Enclave](https://openenclave.io/sdk/),
+including development, debugging, emulators, and deployment!
+
+For more information on the technology,
+[see here](https://github.com/openenclave/openenclave).
+
+Currently, this extension supports the following combinations of project types,
+and operating system and trusted execution environments:
+
+1. Linux Development Machine:
+   1. Standalone Project Targets:
+      1. Linux Host w/. Intel SGX Enclave
```

Comment:
Replace "w/." -> "with"

Reply:
This file is a copy of `README`, so fixed here as well.